src/lib/components/composites/project-components/CreateProjectDialog.svelte

```diff
@@ -0,0 +1,151 @@
+<script lang="ts">
+    import { CirclePlus } from "lucide-svelte";
+    import { Button, buttonVariants } from "$lib/components/primitives/button";
+    import * as Dialog from "$lib/components/primitives/dialog";
+    import * as Alert from "$lib/components/primitives/alert";
+    import Input from "$lib/components/composites/input/Input.svelte";
+    import { Schema } from "$lib/schemas";
+    import { cn } from "$lib/utils/shadcn-helper";
+    import ChipsInput from "$lib/components/composites/input/ChipsInput.svelte";
+    import { onMount } from "svelte";
+    import type { User } from "$lib/model/backend";
+    import { BackendController } from "$lib/controller/backend-controller";
+    import { distance } from "fastest-levenshtein";
+    import { goto } from "$app/navigation";
+    import CircleAlert from "lucide-svelte/icons/circle-alert";
+
+    // at the beginning the dialog should not be open
+    let open: boolean = $state(false);
+
+    let errorOccurred = $state(false);
+
+    let projectNameInput: Input;
+    let membersInput: string[] = $state([]);
+
+    // TODO: check, whether this solution scales as the backend contains hundreds / thousands of users
+    // list of possible members (represented by their E-Mail) that can be invited
+    let possibleMemberEMails: string[] = [];
```

Comment:
Otherwise, it would be `eMail` too
```suggestion
    let possibleMemberEmails: string[] = [];
```